docker-compose/docker-compose.base.yml

```diff
@@ -1,85 +1,118 @@
-version: '3'
-
+version: '3.9'
```

Comment:
why 3.9 ? it doesn't exist
https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element
https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-matrix

Reply:
true 😮
It seem that during a long time [there was confusion](https://forums.docker.com/t/docker-compose-version-3-8-or-3-9-for-latest/102439) ([with inconsistencies in the doc](https://github.com/docker/docs/issues/12214))  
but in the end, [3.9 never existed](https://github.com/docker/docs/pull/17390)